### PR TITLE
PLT-1019: Move to Cardano api value

### PR DESCRIPTION
**Pull request by sjoerdvisscher**
Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Formatting, PNG optimization, etc. are updated
    - [x] Important changes are reflected in changelog.d of the affected packages
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [ ] Reference the ADR in the PR and reference the PR in the ADR (if revelant)
    - [x] Reviewer requested

### CODE REVIEW

File: plutus-ledger/src/Ledger.hs [hunk 1]
```diff
@@ -21,7 +20,8 @@ import Ledger.Orphans ()
 import Ledger.Scripts as Export
 import Ledger.Slot as Export
 import Ledger.Tx as Export
-import Ledger.Value as Export (noAdaValue)
+import Plutus.Script.Utils.Ada (Ada)
```
Comment (berewt) [inline plutus-ledger/src/Ledger.hs:23]: Shouldn't we re-export cardano `Value` instead of plutus `Value`?

**Review by berewt — APPROVED** [review]
LGTM

File: plutus-ledger/src/Ledger/Tx.hs [hunk 2]
```diff
@@ -389,17 +392,17 @@ getCardanoTxProducedReturnCollateral :: CardanoTx -> Map V1.Tx.TxOutRef TxOut
 getCardanoTxProducedReturnCollateral tx = maybe Map.empty (Map.singleton (V1.TxOutRef (getCardanoTxId tx) 0)) $
     getCardanoTxReturnCollateral tx
 
-getCardanoTxTotalCollateral :: CardanoTx -> Maybe V1.Value
-getCardanoTxTotalCollateral = onCardanoTx txTotalCollateral
+getCardanoTxTotalCollateral :: CardanoTx -> Maybe C.Lovelace
+getCardanoTxTotalCollateral = onCardanoTx (undefined . txTotalCollateral)
```
[the remark this answers is not in the document]
Comment (sjoerdvisscher) [reply plutus-ledger/src/Ledger/Tx.hs:396]: Thank you! I vaguely remembered I had created some holes before Christmas but I couldn't find them anymore among all the changes!

File: plutus-contract/test/Spec/golden/traceOutput - pubKeyTransactions.txt [hunk 3]
```diff
@@ -24,15 +24,15 @@
...
                            {inputs:
                            reference inputs:
                            collateral inputs:
                            outputs:
-                             - Value (Map [(,Map [("",5000000)])]) addressed to
+                             - 5000000 lovelace addressed to
                                PubKeyCredential: 80a4f45b56b88d1139da23bc4c3c75ec6d32943c087f250b86193ca7 (no staking credential)
-                           mint: Value (Map [])
-                           fee: Value (Map [])
+                           mint:
```
[the remark this answers is not in the document]
Comment (sjoerdvisscher) [reply plutus-contract/test/Spec/golden/traceOutput - pubKeyTransactions.txt:34]: That's how `renderValuePretty` from `cardano-api` renders an empty value. Seems ok to me: nothing is minted.

**Pull request merged**